about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="css/styles.css">
```

Reply:
Code updated! Thanks for the feedback. I use macOS Monterey and Google Chrome.

Reply:
> Done! Thanks for the feedback again.